tsconfig.json

```diff
@@ -67,5 +67,5 @@
     "skipLibCheck": true
   },
   "include": ["src/**/*"],
-  "exclude": ["node_modules", "!node_modules/@types", "src/__tests__", "src/__seeds__"]
```

Comment:
wouldn't be better to exclude them from ts build from here?  Better pattern?

Reply:
not necessarily. they need to be included in a tsconfig file in order to work with eslint and typing checks. I wish typescript would allow to omit them from a build but i dont believe they have that functionality